### DISCUSSION TITLE
react native expo duplicate class kotlin version jdk8 1.8.10 and  jdk7 1.7.22

### DISCUSSION THREAD

**Discussion by LiorAharon98** [community/community#63706] [Programming Help]
hello 
i am using react native with expo (version 49.0.7)
when i try to build with eas i get error : 
![Screenshot_11](https://github.com/community/community/assets/100975756/080d22a8-4d4e-4517-bb5c-a1cb4f046801)

all the package get this error 
i try in the folder of gradle to add this on dependencies:
     constraints {
        implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.8.0") {
            because("kotlin-stdlib-jdk7 is now a part of kotlin-stdlib")
        }
        implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.0") {
            because("kotlin-stdlib-jdk8 is now a part of kotlin-stdlib")
        }
    }

and
implementation(platform("org.jetbrains.kotlin:kotlin-bom:1.8.0"))

but still i get this errors

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @LiorAharon98 ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
